hq_superset/services.py

```diff
@@ -144,6 +149,35 @@ def to_sql(df, replace=False):
         raise ex
 
 
+def subscribe_to_hq_datasource(domain, datasource_id):
+    hq_client = HQClient.get_by_domain(domain)
+    if hq_client is None:
+        hq_client = HQClient.create_domain_client(domain)
```

Comment:
Nit: can this be a separate function? (maybe something like `get_oauth_client_for_domain`)

Reply:
9f4b053